### DISCUSSION TITLE
Add Markor Play Store to Readme file

### DISCUSSION THREAD

**Discussion by kineita** [gsantner/markor#2157] [General] [answered]
Hi guys, I think we should add Markor Play Store to Readme file to avoid misleading, wrong app from play store.

**Comment by gsantner** [comment, marked as answer]
The official project source is here at the developers place (GitHub releases) and on F-Droid. Markor Google Play release is currently there by me, but it's not always like that.

I generally don't want to depend on Google Play to make Markor available to users. They randomly change rules every few months and take it out if they feel like they don't like it anymore. It's not something I want to give my time in for my private free projects.

Users know where to get the real version and updates from.   
That somebody comes up with adware / spyware infused / fake versions will always be the case for any kind of software.  
Users should be aware what they are installing from whom and where.
